helpers/mathjax.js

```diff
@@ -27,7 +28,10 @@ export class HtmlBlockMathRenderer {
 
 		// If we're opting out of deferred rendering, we need to rely
 		// on the global MathJax install for rendering.
-		if (options.noDeferredRendering) return elem;
+		if (options.noDeferredRendering) {
+			window.MathJax.typeset([elem]);
```

Comment:
We can't rely on initial typesetting any longer, so we need to manually typeset. This might be worth wrapping in a try-catch because things may be weird if `elem` is directly in a component's shadowDOM.

Reply:
Note that this will likely need to be changed to:
```
await window.MathJax.startup.promise;
window.MathJax.typeset([elem]);
```
While it's unlikely that we could possibly sneak in before MathJax is done initializing (the initial typeset is usually what takes so long when its startup runs), there's no reason not to guard against that case here. I feel like we may also want to do the same before `typesetShadow` is invoked below.